File: Sorting algorithm using java/bubbleSort.java
class BubbleSort {
    void bubbleSortFunction(int arr[]) {
        int n = arr.length;
        for (int i = 0; i < n - 1; i++)
            for (int j = 0; j < n - i - 1; j++)
                if (arr[j] > arr[j + 1]) {
                    int temp = arr[j]; // swaping them with each other
                    arr[j] = arr[j + 1];
                    arr[j + 1] = temp;
                }
    }

    /* Prints the array */
    void printArray(int arr[]) {
        int n = arr.length;
        for (int i = 0; i < n; ++i)
            System.out.print(arr[i] + " ");
        System.out.println();
    }

    // Driver method to test above
    public static void main(String args[]) {
        BubbleSort ob = new BubbleSort();
        int arr[] = { 64, 34, 25, 12, 22, 11, 90 };
        System.out.println("\n Before Sorting array with bubble sort \n");
        ob.printArray(arr);
        ob.bubbleSortFunction(arr);
        System.out.println("\nAfter Sorting array with bubble sort\n");
        ob.printArray(arr);
    }
}

File: Sorting algorithms in C++/BubbleSort.cpp
#include <bits/stdc++.h>

using namespace std;

void bubbleSort(int a[],int size)
{
    for(int i=0;i< size-1;i++)
    {
      for(int j= 0; j < size-i-1 ;j++)
      {
          if(a[j] > a[j+1])
          {
              int temp = a[j];
              a[j] = a[j+1];
              a[j+1] = temp;
          }
      }
    }
 }     
   

int main()
{
    int size,a[100];
    cin >> size;
    for(int i=0;i<size;i++) cin>> a[i];
    bubbleSort(a,size);
    for(int i=0;i<size;i++) cout<< a[i]<< " ";
    return 0;
}
